new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Kubernetes.java

```diff
@@ -146,6 +147,41 @@ public static boolean isPodTerminating(String namespace, String domainUid, Strin
     return status;
   }
 
+  /**
+   * Checks if a WebLogic server pod has been patched with an expected image.
+   *
+   * @param namespace Kubernetes namespace in which the pod is running
+   * @param domainUid label that the pod is decorated with
+   * @param podName name of the WebLogic server pod
+   * @param image name of the image to check for
+   * @return true if pod's image has been patched
+   * @throws ApiException when there is an error in querying the Kubernetes cluster
+   */
+  public static boolean podImagePatched(
```

Comment:
we should probably make the container an optional param, pods can have more than one container, and probably will in some tests, e.g. when we have a fluentd sidecar.  if the pod has only one container, it is obvious which one to use.

Reply:
Good point. I added the containerName.